bedrock/firefox/templates/firefox/os/faq.html

```diff
@@ -76,41 +99,60 @@ <h2 data-accordion-role="tab">{{_('Buying Firefox OS')}}</h2>
 
           <dt>{{_('What apps will be available on Firefox OS?')}}</dt>
           <dd>
-          {{_('The growing list includes leading apps like Facebook, Twitter, Nokia HERE maps, Cut the Rope, and Time Out.')}}
-          {{_('Many more are being made available every day.')}}</dd>
+          {% if l10n_has_tag('fxos_faq_updates') %}
+            <p>{{_('The Firefox Marketplace offers an open app environment that leverages the scale of the Web as a development platform; allowing operators, phone manufacturers and developers greater flexibility.')}}</p>
+            <p>
+              {% trans url='https://marketplace.firefox.com/' %}
+              The growing list of apps for Firefox OS includes Pinterest, Facebook, LINE, Twitter, airbnb, EA Games, SoundCloud, outlook.com, Cut the Rope, Dailymotion, goal.com and TomTom Maps. Many more are being made available every day. For more information, visit <a href="{{ url }}">https://marketplace.firefox.com/</a>
+              {% endtrans %}
+            </p>
+          {% else %}
+            {{_('The growing list includes leading apps like Facebook, Twitter, Nokia HERE maps, Cut the Rope, and Time Out.')}}
+            {{_('Many more are being made available every day.')}}
+          {% endif %}
+          </dd>
 
           <dt>{{_('Is Firefox OS available in my language?')}}</dt>
           <dd>
-          {{_('Firefox OS is currently available in Spanish, Polish, English, Italian, Portuguese, German, Serbian, Hungarian and Greek.')}}
-          {{_('We’re working on adding more languages all the time.')}}</dd>
-
+          {% if l10n_has_tag('fxos_faq_updates') %}
+            <p>
+              {% trans link=url('firefox.os.devices') %}
+              Firefox OS is currently available in a number of different languages. Check out the device market availability at the <a href="{{ link }}">the device.</a>
```

Comment:
I think you meant to use `at the <a href="%(link)s">devices page</a>` here, the current link text is wrong.